Backend/stack.yaml

```diff
@@ -81,7 +81,7 @@ extra-deps:
       - app/mock-registry
   # -  ../../shared-kernel/lib/mobility-core
   - git: https://github.com/nammayatri/shared-kernel.git
-    commit: 0655c0de03b86fa11d30e4212af4f11d248928c1
+    commit: 97beaeada83a9cbddced0f34e98846dfd1579cb9
```

Comment:
This is incorrect value. Update it with latest hash of shared-kernel d09348f46006c60ace8c613b75a66ba33fcf9a72